src/main/resources/config/config.yaml

```diff
@@ -0,0 +1,8 @@
+logging:
```

Comment:
maybe we should think about the logging format, and whether we can use JSON logging straight away in this app?